packages/front-end/components/Experiment/EditTargetingModal.tsx

```diff
@@ -80,7 +87,10 @@ export default function EditTargetingModal({
     coverage: lastPhase?.coverage ?? 1,
     hashAttribute: experiment.hashAttribute || "id",
     fallbackAttribute: experiment.fallbackAttribute || "",
-    hashVersion: experiment.hashVersion || 2,
+    hashVersion:
+      experiment.hashVersion || hasSDKWithNoBucketingV2
+        ? 1
+        : (2 as 1 | 2 | undefined),
```

Comment:
Will this work?  `useSDKConnections` starts out as null, makes an API request, and then gets the real list.  Is that too late when using as a form default value?

Reply:
good point

Reply:
turns out its fine(ish) because experiment pages trigger a fetch of sdk-connections on load which then gets cached via useSWR() and is ready once EditTargetingModal appears.

Could possibly make this more resilient with some mutate/useEffect logic, but not sure its worth it...